src/frontend/sharedComponents/CameraView.tsx

```diff
@@ -82,17 +89,58 @@ export const CameraView = ({onAddPress}: Props) => {
 
     setCapturing(true);
 
+    const captureOptions = location
+      ? {
+          ...BASE_CAMERA_CAPTURE_OPTIONS,
+          // Apparently not all devices will encode GPS information when taking a photo
+          // so we provide it manually to ensure its presence.
+          // https://github.com/expo/expo/commit/dce5905664dc4559ea1935a6c946526e4c46278c
+          additionalExif: {
+            GPSLongitude: location.coords.longitude,
+            GPSLongitudeRef: location.coords.longitude >= 0 ? 'E' : 'W',
+            GPSLatitude: location.coords.latitude,
+            GPSLatitudeRef: location.coords.latitude >= 0 ? 'N' : 'S',
+            GPSAltitude:
+              typeof location.coords.altitude === 'number'
+                ? location.coords.altitude
+                : undefined,
+            // TODO: Should we also write `GPSDateStamp` and `GPSTimeStamp
+          } satisfies {
+            [key in Extract<keyof PhotoEXIF, `GPS${string}`>]?: PhotoEXIF[key];
+          },
+        }
+      : BASE_CAMERA_CAPTURE_OPTIONS;
```

Comment:
definitely want some visibility on this. basically, not all devices will add gps-related EXIF fields when taking a photo. See the expo-camera commit that's referenced for some additional context.

instead, we make sure that those fields are written to if we have a location to work with.

need some feedback about whether this is sufficient or if we need to write to more fields, e.g. the gps date stamp and time stamp

Reply:
Do you know if this overwrites the exif data if it does exist? Or does it only insert it when it doesn't exist?

Reply:
Good question: I don't know for sure but my guess is that it would overwrite it, which probably isn't ideal. I did think about this but there's no effective way of making sure that this data exists within the image otherwise, because we can't conditionally apply this before the picture is created.

open to thoughts about the potential overwriting is okay. i think in the case of phones taking photos, i'd be curious to know how many actually include the gps info by default because from what i can gather, it's almost never included.

Reply:
hard to tell from the discussion around it, but it seems like `expo-camera` deliberately omits fields related to the GPS due to "system limitations", so it might be the case that for the fields of interest here, there wouldn't be existing values to overwrite 🤷

Reply:
The more I think about this the more i don't think it actually matters. The point of this feature is to give some validity to our photos. Whether the GPS comes from the "pure" exif data or whether it comes from this `additionalExif` data doesnt really change the validity of the data. We can easily show that the `additionalExif` is still valid data that was collected at the time of taking the photo. And i imagine (though im not actually sure) that the "pure" exif data, if it exists, gets the information from the phones native gps anyway - aka the same source.

Reply:
So yeah I think i would add `GPSDateStamp` and `GPSTimeStamp`

Reply:
Ok, yeah on digging the [gps info is just set via android location](https://developer.android.com/reference/androidx/exifinterface/media/ExifInterface?hl=en#setGpsInfo(android.location.Location)), which is what expo uses, so it doesn't matter if it rewrites it or not

Reply:
yeah generally agree with your assessment! one thing to note is that there's technically room for potential loss of precision when it comes to the GPS info, just because we throttle the location information we receive for usage in the app. it'll always be more precise to retrieve the location on the spot rather than use something that's potentially throttled 😄 

don't think it'll be a huge issue but just noting that for future reference.

> So yeah I think i would add GPSDateStamp and GPSTimeStamp

I was afraid you were going to confirm this 😂 the formats used in the spec are subtly cursed (basically will need to do something like [this](https://github.com/digidem/mapeo-mock-data/blob/25462bdff06d3d71eb38e8403aa12d0ce0c7a3d9/lib/faker.js#L60-L85))